RadarSDK/RadarBeaconManager.m

```diff
@@ -265,21 +265,13 @@ - (CLBeaconRegion *)regionForUUID:(NSString *)uuid {
 }
 
 - (void)handleBeacons {
-    if (self.beaconUUIDs.count == 0 && self.nearbyBeaconIdentifiers.count + self.failedBeaconIdentifiers.count == self.beacons.count) {
```

Comment:
Not sure if this is intentional but this check meant that we did not use ranging to determine beacon enters/exits when using UUIDs